lib/command_runner.dart

```diff
@@ -364,8 +364,10 @@ abstract class Command<T> {
   List<String> get aliases => const [];
 
   Command() {
-    argParser.addFlag('help',
-        abbr: 'h', negatable: false, help: 'Print this usage information.');
+    if (!argParser.allowsAnything) {
```

Comment:
Are we confident this won't break anyone?

Reply:
If I recall the `allowAnything` arg parser never worked with a `Command` anyways. So I think this is likely fine outside of maybe some weird edge cases where somebody implemented `ArgParser`?

Reply:
If they were using the `allowAnything` parser, the `argParser.addFlag()` call below would throw an exception (so, I don't think any existing code will be broken here).